hail/src/main/scala/is/hail/io/tabix/TabixReader.scala

```diff
@@ -117,7 +116,6 @@ class TabixReader(val filePath: String, fs: FS, idxFilePath: Option[String] = No
       fatal(s"Hail only supports tabix indexing for VCF, found format code ${ format }")
     val colSeq = readInt(is)
     val colBeg = readInt(is)
-    val colEnd = readInt(is)
```

Comment:
this is a bug.

Reply:
paging @chrisvittal , can you either push a fix to this branch or explain to where I should thread colend?

Reply:
Or can I just read the int and drop it?

Reply:
just don't bind it to a variable. readInt() is side-effecting in that it moves the cursor. The value isn't needed here